OpenSSL/crypto.py

```diff
@@ -2304,6 +2372,19 @@ def verify(cert, signature, data, digest):
         _raise_current_error()
 
 
+def verify_cert(store_ctx):
+    """
+    Verify a certificate in a context.
+
+    :param store_ctx: The :py:class:`X509StoreContext` to verify.
+    :raises: Error
+    """
+    store_ctx._init()
+    ret = _lib.X509_verify_cert(store_ctx._store_ctx)
+    store_ctx._cleanup()
```

Comment:
I think I'm leaning towards suggestion that `verify_cert` just be a method of `X509StoreContext`.  That way it can be `self._init()` and `self._cleanup()` which is nicer than accessing private methods of a non-self object.

Also possibly the `_cleanup` call should be in a `finally` block - though I can't suggest how to write a unit test that demonstrates that this matters.